### ISSUE TITLE
bug: Windows client stuck reconnecting

### ISSUE THREAD

**Issue by JamesTDG** [revoltchat/desktop#162]
### What happened?

While closing the client via task manager is necessary to temporarily fix this bug, it should be documented. This bug occurs whenever you leave the client open for too long.

Essentially, the client is stuck showing a reconnecting blurb underneath the server name on the channel bar. While it is there, users can still send messages to servers, but are unable to receive messages. Such an example can be seen at message 01H45E3GGXH96HZNJPXTEMATMS, where I sent a message after another one (01H45DS5GVJ1QW7K9QXRBM0BVT), and I could not see any other responses between or afterwards. It was only after I opted to check the browser client that I noticed something was up.

**Comment by Rexogamer** [comment]
Have you tried reloading the app with Control + R?

**Comment by JamesTDG** [comment]
> 

I have not. I will test that out the next time it happens.

**Comment by insertish** [comment]
Superseded by https://github.com/revoltchat/frontend/issues/272
